docs/data/charts/bars/BarAnimation.tsx

```diff
@@ -70,7 +71,7 @@ export default function BarAnimation() {
 const highlightScope = {
   highlight: 'series',
   fade: 'global',
-} as const;
+} as HighlightScope;
```

Comment:
This should be moved to `const highlightScope: HighlightScope = {` else the `as HighlightScope` will coerce the typing